src/core/Akka.Discovery.Tests/Akka.Discovery.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\common.props" />
+  <Import Project="..\..\xunitSettings.props" />
+
+  <PropertyGroup>
+    <AssemblyTitle>Akka.Discovery.Tests</AssemblyTitle>
+    <TargetFrameworks>$(NetFrameworkTestVersion);$(NetCoreTestVersion)</TargetFrameworks>
+    <LangVersion>8</LangVersion>
```

Comment:
I  hope this is fine for new proyects.

Reply:
Yep, that should be fine as long as we don't use default interfaces - which aren't binary backwards compatible